sections/synths/components/SynthPriceCol.tsx

```diff
@@ -3,23 +3,34 @@ import styled from 'styled-components';
 
 import media from 'styles/media';
 
-import { Period } from 'constants/period';
-
 import CurrencyPrice from 'components/Currency/CurrencyPrice';
 import { CurrencyKey } from '@synthetixio/contracts-interface';
 import useSelectedPriceCurrency from 'hooks/useSelectedPriceCurrency';
 import { NO_VALUE } from 'constants/placeholder';
-import useSynthetixQueries from '@synthetixio/queries';
+import useSynthetixQueries, { exchanges, SynthetixQueryContext } from '@synthetixio/queries';
+import React from 'react';
 
 type SynthPriceColProps = {
 	currencyKey: CurrencyKey;
 };
 
 const SynthPriceCol: FC<SynthPriceColProps> = ({ currencyKey }) => {
-	const { useExchangeRatesQuery, useHistoricalRatesQuery } = useSynthetixQueries();
+	const { useExchangeRatesQuery } = useSynthetixQueries();
 
 	const exchangeRatesQuery = useExchangeRatesQuery();
-	const historicalRates = useHistoricalRatesQuery(currencyKey, Period.ONE_DAY);
+
+	const exchangesURL =
+		React.useContext(SynthetixQueryContext)?.context.subgraphEndpoints.exchanges || '';
+	const historicalRates = exchanges.useGetRateUpdates(
+		exchangesURL,
+		{
+			orderBy: 'timestamp',
+			orderDirection: 'desc',
+			where: { timestamp_gt: Date.now() - 86400, currencyKey },
```

Comment:
Rather using a magic number can you add a `PeriodInSeconds` in `constants/period` similar to how we have `PERIOD_IN_HOURS`

Reply:
https://github.com/Synthetixio/staking/pull/769/commits/a51b4cf2e4fb959f0bac4305dfe0f93adedf9e02